Tests/DisabledTests/ClientSessionTests.swift

```diff
@@ -80,7 +80,7 @@ final class ClientSessionTests: MongoSwiftTestCase {
     ]
 
 // This function causes the compiler to crash on older versions of swift due to a bug in the compiler.
-#if swift(>=5.1)
+#if os(OSX) && swift(>=5.1)
```

Comment:
this crash still happens on linux 5.1, so I just made this a macOS only test until we can be sure linux is working.